app/models/host/managed.rb

```diff
@@ -833,6 +833,11 @@ def compute_resource_or_model
 
   private
 
+  # Grouped permissions for STI models are explicitly not allowed
```

Comment:
the comment is a bit misleading, it's not caused by the STI but rather other known permissions introduced by plugins are not mapped correctly

some more context for others, since Host is so exposed object, I think it's reasonable to stop relying on auto detection and simply hardcode resource part of the permission name here. So please reformulate a comment to something like 

`Permissions introduced by plugins for this class can cause resource <-> permission names mapping to fail randomly so as a safety precaution, we specify the name more explicitly`

Then you have my :+1: (I'll wait a bit so others have chance to raise concerns)

Reply:
Sure, done.